### PR TITLE
Bump operator libraries/dependencies

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: operator/model/pom.xml [hunk 1]
```diff
@@ -70,7 +44,6 @@
                 </executions>
                 <configuration>
                     <source>src/main/resources/kubernetes/crd.yml</source>
-                    <extraAnnotations>true</extraAnnotations>
```
Comment (andreaTP) [inline operator/model/pom.xml:73]: giving up on trying to keep the Builders and Fluent API, it's too much of a burden to form a sane classpath with `sundrio`.

File: operator/pom.xml [hunk 2]
```diff
@@ -25,10 +25,13 @@
...
         <awaitility.version>4.2.0</awaitility.version>
-        <kubernetes-client-java-gen.version>6.7.2</kubernetes-client-java-gen.version>
-        <sundrio.version>0.95.0</sundrio.version>
+        <kubernetes-client-java-gen.version>6.8.1</kubernetes-client-java-gen.version>
         <assertj.core.version>3.24.2</assertj.core.version>
-        <quarkus.operator.sdk.version>5.1.4</quarkus.operator.sdk.version>
+        <bouncycastle.version>1.70</bouncycastle.version>
+        <!-- ATTENTION! The Quarkus version needs to be aligned with the Quarkus Operator SDK -->
+        <!-- TODO: possibly stabilize on something shared with the rest of the repo -->
+        <quarkus.version>3.2.0.Final</quarkus.version>
```
Comment (andreaTP) [inline operator/pom.xml:33]: In this module the Quarkus version should be aligned with the one used by the Operator SDK.
@carlesarnal any specific reason for not bumping to 3.2 even in the rest of the project?

Comment (carlesarnal) [reply operator/pom.xml:33]: No reason at all, 3.0.2.Final was just the one available when I worked on the migration.

Comment (andreaTP) [reply operator/pom.xml:33]: let me merge this one first, and we can eventually align 👍 is dependabot configured to bump quarkus core deps?

**Pull request merged**